pkg/minikube/machine/start.go

```diff
@@ -141,8 +141,8 @@ func createHost(api libmachine.API, cfg config.ClusterConfig, n config.Node) (*h
 
 	cstart := time.Now()
 	glog.Infof("libmachine.API.Create for %q (driver=%q)", cfg.Name, cfg.Driver)
-	// Allow two minutes to create host before failing fast
-	if err := timedCreateHost(h, api, 2*time.Minute); err != nil {
+
+	if err := timedCreateHost(h, api, 4*time.Minute); err != nil {
```

Comment:
one of the reasons that this was made lower was to catch 
connecting to SSH...

that meant the SSH keys were not copied to the machine, and no matter how long u try it would not be helpful

how about make this timedCreateHost smarter and fail fast on the things that we can do NOTHING about it
(like SSH problems or network or something like that ?)

Reply:
here is the exact error message if you wanna catch that one to fail fast:
```
I0212 20:18:31.490329   17811 main.go:110] libmachine: Error dialing TCP: dial tcp 192.168.64.44:22: connect: connection refused
I0212 20:18:35.497395   17811 main.go:110] libmachine: Error dialing TCP: dial tcp 192.168.64.44:22: connect: connection refused
I0212 20:18:39.502636   17811 main.go:110] libmachine: Error dialing TCP: dial tcp 192.168.64.44:22: connect: connection refused
I0212 20:18:43.512406   17811 main.go:110] libmachine: Error dialing TCP: dial tcp 192.168.64.44:22: connect: connection refused
```

(maybe this might need a PR on libmachine (not sure))

Reply:
I don't think it's feasible to detect other conditions. Is 3 minutes an acceptable compromise?